LocationList.py

```diff
@@ -262,6 +435,27 @@ def shop_address(shop_id, shelf_id):
     ("GC Shop Item 6",                                               ("Shop",         0x2E,  0x35, (shop_address(8, 5), None),  'Buy Heart',                             ("Goron City", "Shops",))),
     ("GC Shop Item 7",                                               ("Shop",         0x2E,  0x36, (shop_address(8, 6), None),  'Buy Red Potion for 40 Rupees',          ("Goron City", "Shops",))),
     ("GC Shop Item 8",                                               ("Shop",         0x2E,  0x37, (shop_address(8, 7), None),  'Buy Heart',                             ("Goron City", "Shops",))),
+    ("GC Spinning Pot Bomb Drop 1", ("Collectable", 0x62, 0x41, ([0x22A82F4], None), 'Bombs (5)', ("Goron City", "RupeeTower"))),
+    ("GC Spinning Pot Bomb Drop 2", ("Collectable", 0x62, 0x42, None, 'Bombs (5)', ("Goron City", "RupeeTower"))),
+    ("GC Spinning Pot Bomb Drop 3", ("Collectable", 0x62, 0x43, None, 'Bombs (5)', ("Goron City", "RupeeTower"))),
+    ("GC Spinning Pot Rupee Drop 1", ("Collectable", 0x62, 0x44, None, 'Rupee (1)', ("Goron City", "RupeeTower"))),
+    ("GC Spinning Pot Rupee Drop 2", ("Collectable", 0x62, 0x45, None, 'Rupee (1)', ("Goron City", "RupeeTower"))),
+    ("GC Spinning Pot Rupee Drop 3", ("Collectable", 0x62, 0x46, None, 'Rupee (1)', ("Goron City", "RupeeTower"))),
+    ("GC Spinning Pot PoH Drop Rupee 1", ("Collectable", 0x62, 0x47, None, 'Rupee (1)', ("Goron City", "RupeeTower"))),
+    ("GC Spinning Pot PoH Drop Rupee 2", ("Collectable", 0x62, 0x48, None, 'Rupee (1)', ("Goron City", "RupeeTower"))),
```

Comment:
Major tabbing issue

Reply:
I reformatted the entirety of LocationList.py so this should be good now.